Plotting.py

```diff
@@ -456,7 +475,7 @@ def computeStats(data, columns, min, max):
 
 
 if __name__ == '__main__':
-    # Path to the json file, e.g. C:\\Users\\username\\Documents\\Data\\Data.json
+    # Path to the json file, e.g. C:\\Users\\username\\Documents\\Data\\Data.csv
 
     path = ''
```

Comment:
Perhaps add an example path like "./output/all_statistics.csv"?

Reply:
@SylviaWhittle has noticed issues with using relative paths and is working on adding pathlib. It might be better to have her add that example path in her new pull request?